src/connection-manager/index.ts

```diff
@@ -684,23 +719,27 @@ export class DefaultConnectionManager extends EventEmitter<ConnectionManagerEven
       return false
     }
 
+    // check pending connections
+    if (this.incomingPendingConnections === this.opts.maxIncomingPendingConnections) {
```

Comment:
```suggestion
    if (this.incomingPendingConnections >= this.opts.maxIncomingPendingConnections) {
```
?

Reply:
`this.opts.maxIncomingPendingConnections` is optional so it could possibly be undefined, given the current mechanism it would have to equal the threshold before exceeding it.

Reply:
isn't there a maxIncomingPendingConnections default like Infinity?

Reply:
Good point, [it defaults to](https://github.com/maschad/js-libp2p/blob/feat/seperate-limits-for-connections/src/connection-manager/index.ts#L139)  `10` , I've updated.